modules/apps/object/object-test/src/testFunctional/tests/macros/VisualizeObjectDisplayPageTemplate.macro

```diff
@@ -0,0 +1,37 @@
+definition {
+
+	macro createObjectPageTemplate {
+		VisualizeObjectDisplayPageTemplate.openPageTemplate();
+
+		Click(locator1 = "VisualizeObjectDisplayPageTemplate#SELECT_BLANK_OBJECT_DISPLAY_PAGE_TEMPLATE");
+
+		Type(
+			locator1 = "VisualizeObjectDisplayPageTemplate#NAME_OBJECT_DISPLAY_PAGE_TEMPLATE",
+			value1 = "Object Display Page Template");
+
+		Select(
+			locator1 = "VisualizeObjectDisplayPageTemplate#OBJECT_CONTENT_TYPE",
+			value1 = "Object");
+
+		Select(
+			locator1 = "VisualizeObjectDisplayPageTemplate#OBJECT_SUBTYPE",
+			value1 = "User");
+	}
+
+	macro openPageTemplate {
+		Navigator.openSpecificURL(url = "/group/guest/~/control_panel/manage?p_p_id=com_liferay_layout_page_template_admin_web_portlet_LayoutPageTemplatesPortlet&p_p_lifecycle=0&p_p_state=maximized&p_p_mode=view&_com_liferay_layout_page_template_admin_web_portlet_LayoutPageTemplatesPortlet_mvcPath=%2Fselect_display_page_master_layout.jsp&_com_liferay_layout_page_template_admin_web_portlet_LayoutPageTemplatesPortlet_redirect=%2Fgroup%2Fguest%2F~%2Fcontrol_panel%2Fmanage%3Fp_p_id%3Dcom_liferay_layout_page_template_admin_web_portlet_LayoutPageTemplatesPortlet%26p_p_lifecycle%3D0%26p_p_state%3Dmaximized%26p_p_mode%3Dview%26_com_liferay_layout_page_template_admin_web_portlet_LayoutPageTemplatesPortlet_tabs1%3Ddisplay-page-templates%26p_p_auth%3DA1aHfues");
+	}
+
+	macro validateObjectContentType {
+		AssertSelectedLabel(
+			locator1 = "VisualizeObjectDisplayPageTemplate#OBJECT_CONTENT_TYPE",
+			value1 = "Object");
```

Comment:
This value should be passed as a parameter and not inserted directly into the macro.